server/controllers/dashboardController/index.js

```diff
@@ -0,0 +1,44 @@
+import { ObjectId } from 'mongodb'
+import UserModel from '../../models/UserModel'
+import ConfigModel from '../../models/ConfigModel'
+import DashboardService from '../../services/DashboardService'
+
+const DashboardsController = {
+  show: async (req, res) => {
+    try {
+      const { appDb, dataDb } = req.app.locals
+      const { study, subject } = req.params
+
+      const user = await UserModel.findOne(appDb, { uid: req.user })
+      const userConfigurationQuery = {
+        _id: new ObjectId(user.preferences.config),
+      }
+      const config = await ConfigModel.findOne(appDb, userConfigurationQuery)
+      const flatConfig = Object.values(config.config).flat()
+
+      const currentDashboard = new DashboardService(
```

Comment:
this is an instance of the `DashboardService`, what do you think about naming it `dashboardService` to make that more obvious?

```suggestion
      const dashboardService = new DashboardService(
```

Reply:
[21d246e](https://github.com/AMP-SCZ/dpdash/pull/344/commits/21d246eb6479a99f1ed04a88b2054072eebd2f2d)